tutorials/smart-contracts/launch-your-first-project/index.md

```diff
@@ -1,4 +1,23 @@
+---
 title: Launch Your First Project
-nav:
-  - index.md
-  - create-contracts
\ No newline at end of file
+description: Follow a step-by-step guide to creating, deploying, and managing your first smart contract project on Polkadot, from coding to execution.
+template: index-page.html
+---
+
+# Launch Your First Smart Contract Project
+
+Kickstart your journey into smart contract development with this comprehensive guide. Learn how to create, deploy, and interact with contracts on Polkadot.
+
+## Development Pathway
+
+- **Beginner-friendly** – step-by-step instructions suitable for newcomers to smart contract development
+- **Hands-on learning** – practical exercises that build real-world skills
+- **Production-ready** – progress from basic concepts to deployment-ready contracts
+
+## Get Started
```

Comment:
I'd also move this up to the intro. This is a get started section and if you look at it as a standalone section, there are no CTAs here to get started.